app/test_main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+from app.main import can_access_google_page
+from typing import Any
+
+
+def test_can_access_google_page_accessible(mocker: Any) -> None:
+    url = "https://www.google.com"
+
+    mocker.patch("main.valid_google_url", return_value=True)
```

Comment:
Add `app.main....` here and in other patches as well.
```suggestion
    mocker.patch("app.main.valid_google_url", return_value=True)
```